src/webgl/shaders/filters/blur.frag

```diff
@@ -0,0 +1,50 @@
+// Single-pass blur filter, taken from Adam Ferriss' repo of shader examples:
+// https://github.com/aferriss/p5jsShaderExamples/blob/gh-pages/4_image-effects/4-9_single-pass-blur/effect.frag
+
+precision highp float;
+
+// lets grab texcoords just for fun
+varying vec2 vTexCoord;
+
+// our texture coming from p5
+uniform sampler2D tex0;
+uniform vec2 texelSize;
+uniform float filterParameter;
+
+void main() {
+
+  vec2 uv = vTexCoord;
+
+  // a single pass blur works by sampling all the neighbor pixels and averaging them up
+  // this is somewhat inefficient because we have to sample the texture 9 times -- texture2D calls are slow :( 
+  // check out the two-pass-blur example for a better blur approach
+  // get the webcam as a vec4 using texture2D
+
+  // spread controls how far away from the center we should pull a sample from
+  // you will start to see artifacts if you crank this up too high
+  float spread = 4.0;
```

Comment:
lets just simplify this whole block to `float spread = max(0.0, filterParameter);` and then set a default value for the filterParameter in the js that is overridden if explicitly set.